src/matchers/toHaveBeenCalledAfter/index.test.js

```diff
@@ -73,12 +74,6 @@ describe('.toHaveBeenCalledAfter', () => {
 });
 
 describe('.not.toHaveBeenCalledAfter', () => {
-  test('fails when given a first mock has not been called', () => {
```

Comment:
why not this? duplication?

Reply:
Yeah, jest warned me that it was obsolete because it was a duplicate 😊

Reply:
fancy stuff 😅